content/en/llm_observability/exploration_and_correlation/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Exploration and Correlation
+description: Learn about how to explore more of your application in LLM Observability.
+---
+
+## Overview
+
+Explore and analyze your LLM applications in production with tools for querying, visualizing, correlating, and investigating data across traces, clusters, and beyond.
+
+Monitor performance, debug issues, evaluate quality, and secure your LLM-powered systems with unified visibility across traces, metrics, and online evaluations.
+
+### Real-time Performance Monitoring
+
+Monitor your LLM application's operational health with built-in metrics and dashboards:
+
+{{< img src="llm_observability/index/llm_dashboard.png" alt=" This image shows a built‑in dashboard highlighting LLM Observability metrics." style="width:100%">}}
```

Comment:
could we use light mode screenshots instead of dark mode to keep all the screenshots consistent on this page?